packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -149,8 +177,15 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
     if (this.props.parentRef) {
       this._fitParentContainer();
     }
+
+    const margin = { ...this.margins };
+    if (!this._isRtl) {
```

Comment:
add explicit check to apply these changes only for horizontal bar chart with axis

Reply:
Done